contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,18 +1,17 @@
 ### Q4 2025 Objectives
 This quarter we're making feature flags faster, more reliable, and way more automated. That means better (and more!) SDKs, more powerful local evaluation, supporting real-time use cases, and using AI to handle the tedious stuff. We're also talking to a bunch of users to figure out what to build next, fixing the foundations so nothing breaks, and shipping features that make flags easier to wrangle when you have thousands of them.
 
-#### Awesome documentation <TeamMember name="Dylan Martin" photo />
+#### Awesome documentation <TeamMember name="Dylan Martin" photo /> driving, shared across the team
 
 Our docs should be as good as the <SmallTeam slug="experiment" />'s. We're partnering with the <SmallTeam slug="content" /> to refresh everything, writing real-world walkthroughs, and making it easier to jump from the app to the right docs.
 
-#### Make local evaluation the default server-side evaluation strategy <TeamMember name="Dylan Martin" photo /> <TeamMember name="Phil Haack" photo />
+#### Make local evaluation the default server-side evaluation strategy <TeamMember name="Phil Haack" photo />
 
 Local eval means faster, cheaper, and more reliable flags. We want to make it the default for server-side SDKs by:
 - Finishing HyperCache integration to decouple `/local_evaluation` from Postgres
-- Solving the properties problem that's been blocking adoption (it's not very ergonomic to bring your own properties to everything, we'd love to improve this experience)
```

Comment:
cutting this; I think it's a think we solve with better docs (see below), not with some sort of software change